modules/power-monitoring-accessing-dashboards-developer.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-accessing-dashboards-developer_{context}"]
+= Accessing {PM-shortname} dashboards as a developer
+
+You can access {PM-shortname} dashboards from the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
```

Comment:
One more point to add: 
* The user must have view permissions for the project that the dashboard is for. i.e `openshift-power-monitoring`

Reply:
@vprashar2929 I have added this as a point in the Prerequisites.